### ISSUE TITLE
Imaclean Stopped Working

### ISSUE THREAD

**Issue by notoveryet** [sreichel/magento-Defcon2-Imaclean#1]
Hi,

I just upgraded beta version to 1.0 but Imaclean stopped seeing deleted images in my media directory.

I tried reindexing, deleting the core_resource records, re-creating the imaclean table, re-installing the extension, etc.

Still is shows "0 records found" when I click Refresh, even though I know I just deleted an image from a product and it's still in the media directory.

I am running Magento 1.9.2.2 and Imaclean 1.0

Funny thing is, Imaclean beta version worked fine (when the menu was at the top of the Magento menu and not under System).

Thanks in advance!

**Comment by fballiano** [comment]
true, beta version was working, I updated today and it doesn't work anymore

**Comment by fballiano** [comment]
I managed to make it working again changing Helper/Data.php, line 45, with:
$path = Mage::getBaseDir('media') . DS . 'catalog' . DS . 'product';

**Comment by sreichel** [comment]
`$path` has not changed since first release. `substr($path . DS . $entry, 21);` should not work with Mage::getBaseDir('media')?

Any further information?
